src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/ServerType.cs

```diff
@@ -5,9 +5,10 @@ namespace Microsoft.AspNetCore.Server.IntegrationTesting
 {
     public enum ServerType
     {
+        None,
```

Comment:
What the use case for None?

Reply:
So we can tell if a field was accidentally not set. I want to avoid defaulting to an incorrect value.

Reply:
Nullable?